shared/js/background/load.es6.js

```diff
@@ -14,6 +14,30 @@ function url (url) {
     return loadExtensionFile({ url: url, source: 'external' })
 }
 
+async function fetchData (url, fallbackUrl, headers) {
+    let response = null
+    response = await fetch(url, { method: 'GET', headers })
+    if (fallbackUrl && response.status !== 200 && response.status !== 304) {
+        response = await fetch(fallbackUrl, { method: 'GET', headers })
+    }
+
+    return response
+}
+
+function addTestParameters (url) {
+    let newUrl = url
+
+    if (newUrl.indexOf('?') > -1) {
+        newUrl += '&'
+    } else {
+        newUrl += '?'
+    }
+
+    newUrl += 'test=1'
```

Comment:
Why are we sending this parameter?

If we need it, please use the `URL`'s `searchParams` attribute to safely add parameters to a URL.